scss/adminLessonSideNav.module.scss

```diff
@@ -10,35 +10,54 @@ $light-border: hsla(0, 0%, 85%);
 }
 
 .container {
-  background-color: colors.$bg-light-gray;
-  padding: 28px 21px;
-  border-radius: 10px;
   display: flex;
   flex-direction: column;
   align-items: center;
+  padding: 28px 21px;
+  gap: 10px;
+
+  width: 369.01px;
   height: 460px;
-  width: 100%;
-  max-width: 369.01px;
-  row-gap: 23px;
-  justify-content: center;
+
+  background: colors.$secondary;
+
+  box-shadow: 0px 4px 25px rgba(0, 0, 0, 0.07);
+  border-radius: 10px;
 }
 
 .container__heading {
-  font-weight: bold;
+  width: 327px;
+  height: 22px;
+  left: 0px;
+  right: 0px;
+
+  font-weight: 700;
```

Comment:
There seems to be redundant `font-weight` and `font-size` properties.

Reply:
got rid of the extra font-weight and font-size properties

Reply:
Nice!